etc/config.yml.ex

```diff
@@ -38,3 +38,6 @@
 # will be used.
 #websockify_paths:
 #  - /usr/bin/websockify
+#timeout: 5
+# Time spent attempting to connect to a remote desktop before timing out
+# (in seconds)
```

Comment:
Although its really not clear, the comments are supposed to be above the setting that they refer to.  Could you adjust please.